### ISSUE TITLE
feat(Wizard - next): supporting component unit tests

### ISSUE THREAD

**Issue by jeffpuzzo** [patternfly/patternfly-react#7737]
**Depends on**: #7703

Added unit tests for the following:
- WizardFooter
- WizardStep
- WizardToggle
- WizardBody

Note:
- WizardComposableContext, although is implicitly somewhat tests via WizardComposable tests, since more changes are planned for it, any explicit tests will be deferred to a separate task.

In addition:
- Bubbled up a prop for controlling the mount/unmount state of inactive steps to WizardComposable as well.
